backend_kivy.py

```diff
@@ -995,6 +996,39 @@ def _get_style_dict(self, rgbFace):
         return attrib
 
 
+class TimerKivy(TimerBase):
+    '''
+    Subclass of :class:`backend_bases.TimerBase` that uses Kivy for timer events.
+    Attributes:
+    * interval: The time between timer events in milliseconds. Default
+        is 1000 ms.
+    * single_shot: Boolean flag indicating whether this timer should
+        operate as single shot (run once and then stop). Defaults to False.
+    * callbacks: Stores list of (func, args) tuples that will be called
+        upon timer events. This list can be manipulated directly, or the
+        functions add_callback and remove_callback can be used.
+    '''
+    def _timer_start(self):
+        # Need to stop it, otherwise we potentially leak a timer id that will
+        # never be stopped.
+        self._timer_stop()
+        self._timer = Clock.schedule_interval(self._on_timer, self._interval/1000)
```

Comment:
This should be `self._interval / 1000.`. The period is required in py3, otherwise it'd be integer division. Also, there should be spaces around the forward slash.